imap/ical_support.h

```diff
@@ -203,14 +203,6 @@ extern void icalcomponent_normalize_x(icalcomponent *ical);
 #define icalcomponent_get_acknowledged_property(comp) \
     icalcomponent_get_first_property(comp, ICAL_ACKNOWLEDGED_PROPERTY)
 
-#ifndef HAVE_RFC7986_COLOR
-
-/* Replacement for missing function in 3.0.0 <= libical < 3.0.5 */
-
-extern icalproperty *icalproperty_new_color(const char *v);
-
-#endif /* HAVE_RFC7986_COLOR */
-
```

Comment:
Should we also remove the implementation of this from [ical_support.c](https://github.com/cyrusimap/cyrus-imapd/blob/master/imap/ical_support.c#L2121-L2132), or does it need to remain for some reason I don't understand?

Reply:
I wound up removing it myself!